backend/app/services/knowledge/summary_service.py

```
@@ -0,0 +1,572 @@
+        Returns:
+            Dict with status and message
+
+        Raises:
+            ValueError: If document not found or not accessible
+        """
+        if not settings.SUMMARY_ENABLED:
+            return {"status": "disabled", "message": "Summary feature is disabled"}
+
+        doc = KnowledgeService.get_document(db, document_id, user_id)
+        if not doc:
+            raise ValueError("Document not found or access denied")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)